code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -133,15 +133,14 @@
 			fire_cone.StartCooldown(0)
 			fire_cone.Trigger(target = target)
 			meteors.StartCooldown(0)
-			INVOKE_ASYNC(meteors, /datum/action/proc/Trigger, target)
+			meteors.Trigger(target = target)
```

Comment:
why're we changing this away from invoking async?

Reply:
doesntw ork